unity-client/Assets/ABConverter/Utils.cs

```diff
@@ -0,0 +1,383 @@
+using DCL.Helpers;
+using GLTF.Schema;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.Networking;
+using static DCL.ContentServerUtils;
+
+namespace DCL
+{
+    public abstract partial class ABConverter
+    {
+        public static class PathUtils
+        {
+            /// <summary>
+            /// Gets the relative path ("..\..\to_file_or_dir") of another file or directory (to) in relation to the current file/dir (from)
+            /// </summary>
+            /// <param name="to"></param>
+            /// <param name="from"></param>
+            /// <returns></returns>
+            public static string GetRelativePathTo(string from, string to)
+            {
+                var fromPath = Path.GetFullPath(from);
+                var toPath = Path.GetFullPath(to);
+
+                var fromUri = new Uri(fromPath);
+                var toUri = new Uri(toPath);
+
+                var relativeUri = fromUri.MakeRelativeUri(toUri);
+                var relativePath = Uri.UnescapeDataString(relativeUri.ToString());
+
+                string result = FixDirectorySeparator(relativePath);
+
+                return result;
+            }
+
+            /// <summary>
+            /// 
+            /// </summary>
+            /// <param name="fullPath"></param>
+            /// <returns></returns>
+            public static string FullPathToAssetPath(string fullPath)
+            {
+                char ps = Path.DirectorySeparatorChar;
+
+                fullPath = fullPath.Replace('/', ps);
+                fullPath = fullPath.Replace('\\', ps);
+
+                string pattern = $".*?\\{ps}(?<assetpath>Assets\\{ps}.*?$)";
+
+                var regex = new Regex(pattern);
+
+                var match = regex.Match(fullPath);
+
+                if (match.Success && match.Groups["assetpath"] != null)
+                    return match.Groups["assetpath"].Value;
+
+                return string.Empty;
+            }
+
+            public static string FixDirectorySeparator(string path)
+            {
+                char ps = Path.DirectorySeparatorChar;
+                path = path.Replace('/', ps);
+                path = path.Replace('\\', ps);
+                return path;
+            }
+
+            /// <summary>
+            /// 
+            /// </summary>
+            /// <param name="assetPath"></param>
+            /// <returns></returns>
```

Comment:
You forgot to write a description here :)

Reply:
oops